examples/library/src/Server.hs

```diff
@@ -108,33 +109,37 @@ libraryServer conn = resolver
     bookId :: Entity Book -> ServerErrorIO Integer
     bookId (Entity (BookKey k) _) = pure $ toInteger k
     bookTitle :: Entity Book -> ServerErrorIO T.Text
-    bookTitle (Entity _ Book { bookTitle }) = pure bookTitle
+    bookTitle (Entity _ Book { bookTitle = tl }) = pure tl
```

Comment:
I think it looks simpler with `NamedFieldPuns`, what do you think? 😕

Reply:
100% agree. I changed them because having unique names help me visualise where everything is coming from. I'll send a fix in a few minutes.